GetStream/stream-chat-swift
# Fix file attachments not rendering file size

### 🔗 Issue Links
Resolves https://github.com/GetStream/ios-issues-tracking/issues/679

### 🎯 Goal
Fix file attachments not rendering file size when loading the attachments from the server.

### 🛠 Implementation
This issue was happening because we were not rendering the file size the local `uploadingState` was nil. 

### 🧪 Manual Testing Notes
1. Upload a file
2. Should show the size of the file.
3. Logout
4. Login
5. Load the same file from the server
6. It should still show the size of the file.

### ☑️ Contributor Checklist
